specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json

```diff
@@ -10773,6 +11343,42 @@
       },
       "x-ms-parameter-location": "method"
     },
+    "EntitiesQueriesKindParameter": {
```

Comment:
Why not use "EntityQueryKind"?
we have the Expansion there already

Reply:
Exactly. I my options are only "Activity" and "Insights". That why I cannot reuse it.